spec/network_x/traversals/edge_dfs_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+RSpec.describe NetworkX::DiGraph do
+  subject { graph }
+
+  let(:graph) { described_class.new(type: 'directed') }
+
+  before do
+    graph.add_nodes(%w[A B])
+    graph.add_edge('A', 'B')
+    graph.add_edges([%w[B A], %w[B D]])
+    graph.add_node('E')
+  end
+
+  context 'when dfs_edges is called' do
+    subject { NetworkX.edge_dfs(graph, 'A') }
+
+    it do
+      is_expected.to eq([["A", "B"], ["B", "A"], ["B", "D"]])
+    end
```

Comment:
It kinda feels weird to read an `it do` condition without any description (I know that's been covered by the context). `it { is_expected.to eq(something)}` reads more naturally than a `do-end` block.

Has this been done for rubocop?

Reply:
No, I haven't done any offense removal till now. Will do it shortly.